README.md

```diff
@@ -7,180 +7,211 @@ gollum -- A wiki built on top of Git
+1. There are still some bugs and this setup is not ready for production yet. You can track the progress at #1044.
 
 ## INSTALLATION
 
-The best way to install Gollum is with RubyGems:
-
-```bash
-$ [sudo] gem install gollum
-```
-You may first need to install some additional [development packages](https://github.com/gollum/gollum/wiki/Installation) for your OS.
-
-If you're installing from source, you can use [Bundler][bundler] to pick up all the
-gems:
-
-```bash
-$ bundle install
-```
-
-In order to use the various formats that Gollum supports, you will need to
-separately install the necessary dependencies for each format. You only need
-to install the dependencies for the formats that you plan to use.
-
-* [AsciiDoc](http://asciidoctor.org) -- `gem install asciidoctor`
-* [Creole](http://www.wikicreole.org/) -- `gem install creole`
-* [Markdown](http://daringfireball.net/projects/markdown/) -- `gem install redcarpet`
-* [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown) -- `gem install github-markdown`
-* [Org](http://orgmode.org/) -- `gem install org-ruby`
-* [Pod](http://search.cpan.org/dist/perl/pod/perlpod.pod) -- `Pod::Simple::HTML` comes with Perl >= 5.10. Lower versions should install Pod::Simple from CPAN.
+Varies depending on operating system, package manager and Ruby installation. Generally, you should first install Ruby and then Gollum.
+
+1. Ruby is best installed either via [RVM](https://rvm.io/) or a package manager of choice.
+2. Gollum is best installed via RubyGems:  
+	```
+	[sudo] gem install gollum
+	```
+
+Installation examples for individual systems can be seen [here](https://github.com/gollum/gollum/wiki/Installation).
+
+**Notes:**  
+* Whichever Ruby implementation you're using, Gollum ships with the appropriate default git adapter. So the above installation procedure is common for both MRI and JRuby.
+* If you're installing from source:
+	* Optionally uninstall any previous versions of Gollum:  
+		```
+		[sudo] gem uninstall -aIx gollum
+		```
+	* Install [Bundler](http://bundler.io/).
+	* Navigate to the cloned source of Gollum.
+	* Install dependencies:  
+		```
+		[sudo] bundle install
+		```
+	* And finally:  
+		```
+		rake build
+		[sudo] gem install --no-ri --no-rdoc pkg/gollum*.gem
+		```
+
+### Markups
+
+Gollum presently ships with support for the following markups:
+* [Markdown (Kramdown flavour)](http://kramdown.gettalong.org/quickref.html)
 * [RDoc](http://rdoc.sourceforge.net/)
-* [ReStructuredText](http://docutils.sourceforge.net/rst.html) -- `easy_install docutils`
-* [Textile](http://en.wikipedia.org/wiki/Textile_(markup_language)) -- `gem install RedCloth`
-* [MediaWiki](http://www.mediawiki.org/wiki/Help:Formatting) -- `gem install wikicloth`
 
-[bundler]: http://gembundler.com/
+As denoted by [github/markup](https://github.com/github/markup/blob/master/lib/github/markups.rb), support for additional markups can easily be added by additional installation:
```

Comment:
Okay, how about this:
> Since all markups are rendered by the [github-markup](https://github/com/github/markup) gem, you can easily add support for other markups by additional installation: